tests/utilities/test_as_batch.py

```diff
@@ -0,0 +1,27 @@
+"""Tests of as_batch() and of the _resources parameter tunnel."""
+
+import unittest
+import unittest.mock
+
+import tiledb.cloud.utilities._common
+from tiledb.cloud.utilities._common import as_batch
+
+
+@unittest.mock.patch.object(tiledb.cloud.utilities._common.dag, "DAG")
+def test_propagate_resources(mock_dag):
+    """Assert that as_batch propagates resource parameters."""
```

Comment:
This test shows that `as_batch()` is doing it's job: passing a key parameter that Cloud-UI needs, but doesn't know about, on to cloud functions.